### PR TITLE
Raise error on allocation failure & style nits

### PR DESCRIPTION
**Pull request by FdaSilvaYY**
- Raise a few more error on some allocation failure.
- Fix a few typos.
- Fix some style nits.

##### Checklist

- [ ] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

**Review by paulidale — APPROVED** [review]

File: crypto/bn/bn_mod.c [hunk 1]
```diff
@@ -56,9 +56,13 @@ int bn_mod_add_fixed_top(BIGNUM *r, const BIGNUM *a, const BIGNUM *b,
     if (bn_wexpand(r, mtop) == NULL)
         return 0;
 
-    if (mtop > sizeof(storage) / sizeof(storage[0])
-        && (tp = OPENSSL_malloc(mtop * sizeof(BN_ULONG))) == NULL)
-        return 0;
+    if (mtop > (sizeof(storage) / sizeof(storage[0])) {
```
Comment (t8m) [inline crypto/bn/bn_mod.c:59]: Superfluous `(` added.

Comment (FdaSilvaYY) [reply crypto/bn/bn_mod.c]: Fixup pushed

**Review by t8m — APPROVED** [review]

**Comment by FdaSilvaYY** [comment]
Ping @paulidale : this PR needs your re-approval, i guess ?

**Review by paulidale — APPROVED** [review]

**Review by slontis — APPROVED** [review]

**Comment by paulidale** [comment]
Merged to master, thanks for the fixes.

**Pull request closed without merging**